peakdet/__init__.py

```diff
@@ -34,3 +36,5 @@
 
 __version__ = get_versions()["version"]
 del get_versions
+
+logger.disable("peakdet")
```

Comment:
So that the loguru logger is disabled by default for library users, so they can enable it explicitly if they want to. This is not the case for the CLI however